### PR TITLE
[incubator/openstack-ironic-standalone] Add Openstack Ironic Standalone

### PR DESCRIPTION
**Pull request by jakub-d**
Add a new chart that installs Openstack Ironic in standalone mode.

### CODE REVIEW

File: incubator/openstack-ironic-standalone/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: openstack-ironic-standalone
+version: 0.0.1
```
Comment (giacope) [inline incubator/openstack-ironic-standalone/Chart.yaml]: In `Semantic Versioning` (and therefore in `helm/charts`) the first version needs to be `0.1.0`

Comment (jakub-d) [reply incubator/openstack-ironic-standalone/Chart.yaml]: Thank you - I changed that to `0.1.0`

**Comment by jakub-d** [comment]
@lachie83 - can you please review this PR?

**Comment by jakub-d** [comment]
@linki, @mgoodness, @prydonius, @sameersbn, @seanknox, @viglesiasce, @foxish, @unguiculus, @scottrigby, @mattfarina, @davidkarlsen, @paulczar - I would like to add this chart to the incubator. Can anyone please review it?

I am asking, because we use this chart actively in 2 projects and we would like to maintain the current code in this repository - not on our internal git servers. I guess other people would be interested in that.
It has been waiting for 2 weeks for review.

**Comment by jakub-d** [comment]
In case anyone is interested in Openstack Ironic Standalone, I created a dedicated Helm Repository:

https://github.com/jakub-d/openstack-ironic-standalone

```
helm repo add ironic https://ironic.storage.googleapis.com
helm install ironic/openstack-ironic-standalone -f my-site.yaml
```

I know understand why opensource community complains about Helm Charts review process.

**Pull request closed without merging**